packages/bootstrap-4/test/Form.test.tsx

```diff
@@ -199,37 +199,42 @@ describe("single fields", () => {
   });
 });
 
-describe("Form", () => {
-  test("Renders with appropriate refs", () => {
-    function FormWithRef() {
-      const ref = useRef<
-        CoreForm<{
-          field: string;
-        }>
-      >(null);
-    
-      return (
-        <Form<{
-          field: string;
-        }>
-          ref={ref}
-          formData={{
-            field: "",
-          }}
-          schema={{
-            type: "object",
-            properties: {
-              field: {
-                type: "string",
-              },
+describe("refs", () => {
+  test("Renders with a ref and the Form Element accepts a generic formData argument", () => {  
+    let ref = React.createRef<CoreForm<{
+      field: string;
+    }>>()
+ 
+    renderer.create(
+      <Form<{
+        field: string;
+      }>
+        ref={ref}
+        formData={{
+          field: "",
+        }}
+        schema={{
+          type: "object",
+          properties: {
+            field: {
+              type: "string",
             },
-          }}
-        />
-      );
-    }
+          },
+        }}
+      />
+    );
     
-    const el = renderer.create(<FormWithRef />).toJSON();
-
-    expect(el).toMatchSnapshot();
+    expect(ref.current).toBeDefined();
+    if (ref.current) {
+      expect(ref.current.submit).toBeDefined();
+      expect(ref.current.formElement).toBeDefined();
+      expect(ref.current.validate).toBeDefined();
+    } else {
+      /**
+       * If we hit this, then the ref wasn't properly
+       * forwarded
+       */ 
+      expect(false).toBe(true)
+    }
```

Comment:
removed those extra snapshots, and using these we can also validate a few properties that should be defined on the ref